components/dashboard/src/WhatsNew.tsx

```diff
@@ -41,35 +40,34 @@ export function WhatsNew(props: { visible: boolean, onClose: () => void }) {
+            <p className="pb-2 text-gray-500 text-sm">We have made some layout changes on the dashboard to improve the overall user experience of Gitpod.</p>
         </div>
         <div className="border-t border-b border-gray-200 -mx-6 px-6 py-4">
             <p className="pb-2 text-gray-900 text-base font-medium">VS Code</p>
             <p className="pb-4 text-gray-500 text-sm">We are changing the default IDE to VS Code.</p>
             <ol className="pb-2 text-gray-500 text-sm list-outside list-decimal space-y-2">
                 <li className="ml-5">
                     <div>
-                        <p className="text-gray-500 text-sm">We're preserving all <span className="font-bold">user settings and extensions</span>.</p>
-                        <p className="text-gray-400 text-sm">Extensions you have manually uploaded are not transfered. You'll need to search and install those extensions through the extension panel in VS Code.</p>
+                        <p className="text-gray-500 text-sm">We're preserving most <span className="font-bold">user settings and extensions</span>.</p>
+                        <p className="text-gray-400 text-sm">Extensions you have manually uploaded are not transferred. You'll need to search and install those extensions through the extension panel in VS Code.</p>
                     </div>
                 </li>
                 <li className="ml-5">
                     <div>
                         <p className="text-gray-500 text-sm">We've reduced the number of <span className="font-bold">pre-installed extensions</span>.</p>
-                        <p className="text-gray-400 text-sm">The Theia-based editor included pre-installed extensions for the most popular programming languages which was convenienvt for starters but added additional bloat. You can now install any extensions you need and leave out those you don't.</p>
+                        <p className="text-gray-400 text-sm">The Theia-based editor included pre-installed extensions for the most popular programming languages which was convenient for starters but added additional bloat. You can now install any extensions you need and leave out those you don't.</p>
                     </div>
                 </li>
                 <li className="ml-5">
                     <div>
-                        <p className="text-gray-500 text-sm">You can still <span className="font-bold">switch the editor</span> to Theia.</p>
-                        <p className="text-gray-400 text-sm">In case you run into trouble with the new IDE, you can go to the settings and switch back to the Theia-based editor.</p>
+                        <p className="text-gray-500 text-sm">You can still <span className="font-bold">switch the IDE</span> back to Theia.</p>
+                        <p className="text-gray-400 text-sm">In case you run into trouble with VS Code, you can go to the settings and switch back to the Theia.</p>
```

Comment:
Optional: "settings" might be a bit ambiguous here, given that we're talking mostly about IDEs (but here it's Gitpod's dashboard settings, not the IDE settings).

Maybe "in Gitpod's settings"?